react/react2/week1/todo-list-app/src/App.js

```diff
@@ -39,124 +43,28 @@ const App = () => {
       setIsEmpty(true)
     }
   }, [query, url])
+
+  const contextValue = {
+    inputHandler: inputHandler,
+    users: users,
+    title: 'GitHub user searcher',
```

Comment:
I don't think this is necessary. The title won't change. And you might need errors, or in general fetch status, which can be: loading, data, error